pkg/defkinds/php/composer_test.go

```diff
@@ -35,25 +34,18 @@ func initSuccessfullyLoadAndParseComposerLockTC(
 	).Return(raw, nil)
 
 	return loadComposerLockTC{
-		initial: php.StageDefinition{
-			Dev: true,
-		},
 		context: &builddef.Context{
 			Type:   builddef.ContextTypeLocal,
 			Source: "context",
 		},
 		solver: solver,
-		expected: php.StageDefinition{
-			Dev: true,
-			LockedPackages: map[string]string{
```

Comment:
no longer need to specify which are the locked packages ?

Reply:
It has never been really useful and it takes too much place in the lockfile ^^ Originally, this was meant to ensure the extensions inferred from composer files were not changed but actually that would require re-reading composer files at each and every build. Moreover, the whole set of php packages isn't needed to do so :slightly_smiling_face: